api/CountQueuingStrategy.json

```diff
@@ -125,10 +125,10 @@
               "version_added": "43"
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
`new CountQueuingStrategy({ highWaterMark: 10 })` works for me in Safari.

The result of https://mdn-bcd-collector.appspot.com/tests/api/CountQueuingStrategy/CountQueuingStrategy in Safari 14 is null, so why was this entry set to false? Is the update-bcd script now treating null as false? If so I'll need to pay a lot more attention to any false values being added.

Reply:
I've filed https://github.com/foolip/mdn-bcd-collector/issues/883 about this.

Reply:
Since the constructor was there since the interface was added in https://trac.webkit.org/changeset/188127/webkit, I'll update this to match the version when the interface was exposed.